Partition (WBP)/Lists/LI0039.lsts.yml

```diff
@@ -15,7 +15,7 @@
       Type:               Process
 
   - VI0039A:
-      Type:               Funciton
+      Type:               Function
```

Comment:
This change broke the list. Do not edit the YML files. Please back this change out.

Reply:
done